homeassistant/components/advantage_air/climate.py

```diff
@@ -110,32 +113,32 @@ class AdvantageAirAC(AdvantageAirClimateEntity):
     _attr_hvac_modes = AC_HVAC_MODES
     _attr_supported_features = SUPPORT_TARGET_TEMPERATURE | SUPPORT_FAN_MODE
 
-    def __init__(self, instance, ac_key):
+    def __init__(self, instance: advantage_air, ac_key: int) -> None:
```

Comment:
ac_key is a string
https://github.com/home-assistant/core/blob/dev/tests/components/advantage_air/fixtures/getSystemData.json#L3